scanpy/datasets/_datasets.py

```diff
@@ -386,7 +399,9 @@ def visium_sge(
     ----------
     sample_id
         The ID of the data sample in 10x’s spatial database.
-
+    add_image_path
```

Comment:
I think this could have a better name. Maybe `download_tiff` or `include_tiff`?

Reply:
yes, I'll change it to `download_tif`. Or is there a preference for using "tiff" over "tif"?

Reply:
since image can be non-tif, I would choose a different name

Reply:
maybe `download_tissue_image`?